packages/server/src/middleware/nuxt.js

```diff
@@ -22,6 +22,7 @@ export default ({ options, nuxt, renderRoute, resources }) => async function nux
     } = result
 
     if (redirected) {
+      nuxt.callHook('render:headersSet', url, result, context)
```

Comment:
Actually, this one could not work since the response is ended here: https://github.com/nuxt/nuxt.js/blob/dev/packages/vue-app/template/server.js#L40

Reply:
Interesting.. 
Does it mean I should add it in the file you mentionned ? 
Or just remove it ?